schemas/panels/time-series/mig.cuepart

```diff
@@ -7,12 +7,20 @@ if #panel.type == "timeseries" || #panel.type == "graph" {
             	  	if #panel.options.legend.placement == "bottom" { "Bottom" },
             	  	if #panel.options.legend.placement == "right" { "Right" },
             	  ][0]
```

Comment:
This is my first time changing these mig.cuepart files, and I wasn't 100% sure how to test that they work. Any pointers?

Reply:
Aha, I think I found some failing go tests that answer my question here.